.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_gem:
+  govuk-lint: "configs/rubocop/all.yml"
+
+AllCops:
+  TargetRubyVersion: 2.6
```

Comment:
I don't think we need this [given that Rubocop](https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md#setting-the-target-ruby-version) uses [.ruby-version](https://github.com/alphagov/govuk-docker/blob/master/.ruby-version)

Reply:
https://travis-ci.org/alphagov/govuk-docker/builds/553761983 🤷‍♂

Reply:
🤷‍♂

Reply:
I've found the problem, the gems are vendored and Rubocop is checking all of them

vendoring: https://travis-ci.org/alphagov/govuk-docker/builds/553761983#L565
all: https://travis-ci.org/alphagov/govuk-docker/builds/553761983#L650